docs/snippets/common/storybook-vite-builder-svelte-plugin.js.mdx

```diff
@@ -3,7 +3,7 @@
 
 const preprocess = require('svelte-preprocess');
```

Comment:
require

Reply:
But also, this snippet doesn't seem right...  But it is, I checked the way the svelte-webpack framework works. 
 It means it won't be possible to supply `builder` options to this framework, because they're all sent to the svelte-loader.  @JReinhold I think we need to change this quickly, especially since it'll be a breaking change.